Containers/postgresql/Dockerfile

```diff
@@ -1,36 +1,31 @@
 # From https://github.com/docker-library/postgres/blob/master/15/alpine/Dockerfile
 FROM postgres:15.2-alpine
 
-RUN apk add --no-cache bash openssl shadow grep mawk
+COPY --chmod=775 start.sh /usr/bin/start.sh
+COPY --chmod=775 healthcheck.sh /usr/bin/healthcheck.sh
```

Comment:
```suggestion
COPY --chmod=775 start.sh /start.sh
COPY --chmod=775 healthcheck.sh /healthcheck.sh
```

Reply:
why? the old scripts are also placed in /usr/bin?

Reply:
We could change it then to /start.sh ? I think maybe doing so is more consistant?